x-pack/plugins/maps/public/connected_components/layer_panel/layer_settings/_attribution_popover.scss

```diff
@@ -1,3 +1,3 @@
 .mapAttributionPopover {
-  width: $euiSizeXXL * 12;
+  width: $euiSizeXXL * 8;
```

Comment:
Can we keep the original size? URLs can get very long and having more horizontal space makes the text input more usable.

Reply:
Yes, I'll try to add some styles to make it more similar to the `columnCompressed` form row. I'll also try to remove the horizontal line.